packages/flex-plugin-scripts/src/scripts/check-start.ts

```diff
@@ -182,6 +193,47 @@ export const _checkPluginCount = () => {
   }
 };
 
+/**
+ * First, touch ~/.twilio-cli/flex/plugins.json if it does not exist
+ * Next, checks if this plugin is in this config file. If not, add it.
+ * @private
+ */
+export const _checkPluginConfigurationExists = async() => {
+  // check if plugin.json exists
+  if (!checkFilesExist(paths.cli.pluginsJsonPath)) {
+      mkdirpSync(paths.cli.flex);
+      appendFileSync(paths.cli.pluginsJsonPath, JSON.stringify({plugins: []}, null, 2));
+  }
+
+  // templated read of package.json
+  const config = readJsonFile<CLIFlexConfiguration>(paths.cli.pluginsJsonPath);
+  const newPlugin = config.plugins.find((plugin) => plugin.name === paths.app.name);
```

Comment:
Nitpick: call this `plugin` instead and the variable inside the `.find`, call that just simply `p`